src/__tests__/patients/new/NewPatient.test.tsx

```diff
@@ -1,11 +1,11 @@
 import '../../../__mocks__/matchMediaMock'
 import React from 'react'
-import { mount } from 'enzyme'
-import { Router, MemoryRouter } from 'react-router-dom'
-import { Provider } from 'react-redux'
-import { mocked } from 'ts-jest/utils'
-import { createMemoryHistory } from 'history'
-import { act } from 'react-dom/test-utils'
```

Comment:
Have our prettier rules changed? Why these spaces are removed?

Reply:
I'll look into it, it could be that my IDE is ignoring the eslint rules when formatting the code.

Reply:
What IDE do you use?

Reply:
> What IDE do you use?

Intellij idea ultimate

Reply:
Running `yarn lint` should automatically format. 

@fox1t @tehKapa we may want to put this in a pre commit hook.

Reply:
Also, we need to investigate why this isn't failing the build.

Reply:
@tehKapa do you have any ideas why this is not failing or automatically formatting?

Reply:
mmmh no